frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -67,6 +67,8 @@
 
         vm.subErrors = {};
 
+        vm.privateLeaderboardUnique = false;
```

Comment:
Can you please change the name of the variable to `vm.isChallengeLeaderboardPrivate = false`